File: 01_Leetcode_FAANG/concatenation_of_array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> v;
        v = nums;
        for(int i =0 ; i< nums.size();i++){
            v.push_back(nums[i]);
        }
        return v;
    }
};

File: 01_Leetcode_FAANG/Shubhra/Concatenation_of_Array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] arr = new int[2*n];
        for(int i=0;i<n;i++){
            arr[i]=nums[i];
            }
        int j=0;
        for(int i=n;i<2*n;i++){
            arr[i]=nums[j];
            j++;
            }
        return arr;
        
    }
}
